### DISCUSSION TITLE
Weird Signal

### DISCUSSION THREAD

**Discussion by Hotshot7966** [AlexandreRouma/SDRPlusPlus#1426] [General]
Hi everyone,

I'm using an RTL SDR v3 and a weird signal is always present at the center of the listening window at full zoom out, on an M1 MacBook Pro with v1.2.0.

I have a recording but it won't let me attach it, I've got a couple of screenshots of the waterfall below.

<img width="1432" alt="Screenshot 2024-06-16 at 19 32 53" src="https://github.com/AlexandreRouma/SDRPlusPlus/assets/141958370/11eda669-94d7-4b03-ab4c-81dd6643d4ad">
<img width="1432" alt="Screenshot 2024-06-16 at 19 32 40" src="https://github.com/AlexandreRouma/SDRPlusPlus/assets/141958370/e3b51b04-1534-4af4-9259-2b8ee9751bc9">

Thanks,
Charlie

**Comment by KubaPro010** [comment]
That's the IQ spike at the center, turn on `IQ Correction` to get rid of it, you can also use the correction to get rid of carrier waves that are much stronger than the signal
